examples/eks/cluster.yaml

```diff
@@ -40,7 +40,7 @@ rules:
   - apiGroups:
       - scylla.scylladb.com
     resources:
-      - clusters
+      - scyllaclusters
```

Comment:
This file is a copy and paste from the one in config, no? It seems to have some arbitrary dependencies:

````
$ diff -u  ./config/manager/cluster/cluster.yaml examples/eks/cluster.yaml
... 
apiVersion: scylla.scylladb.com/v1alpha1
 kind: Cluster
 metadata:
-  name: cluster
-  namespace: scylla-manager
+  name: scylla-cluster
+  namespace: scylla
```

Reply:
`cluster.yaml` files are not generated. These are examples what user should provide and these were manually crafted or copied from generic if they suits.